lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -50,7 +50,8 @@
     NOT_ENROLLED_IN_COURSE,
```

Comment:
Adding a comment here, there are some tests that only required changes from None -> {}. Why was there a need to update the pre-existing tests in terms of the data being generated(extra submit by a student etc)?

Reply:
Good question. 

Actually, the value which I have changed from `None` to `{}` is `task_input`. The `task_input` was not used in the previously, hence passing `None` as `task_input` had worked. Now, As we are using it and calling `.get` method, it makes sense to change all existing tests to pass the right falsy value.

Reply:
@awaisdar001 I am able to understand the change from None -> {}. But I can't comprehend why some tests were updated unnecessarily(like adding another submission and then updating expected data)?

Reply:
Mainly talking about lms/djangoapps/instructor_task/tests/test_tasks_helper.py file changes

Reply:
> Mainly talking about lms/djangoapps/instructor_task/tests/test_tasks_helper.py file changes

Sure, I just added the comment and explained why I added new submissions.